src/main/webapp/index.html

```diff
@@ -1,29 +1,167 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8" />
-    <title>TEAM-23 PROJECT</title>
-    <link rel="stylesheet" href="style.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <header>
-      <div class="navbar">
-        <a href="#">TITLE</a>
-        <a href="#">HOME</a>
-        <a href="#">LINK2</a>
-        <a href="#">LINK3</a>
-        <div class="search-container">
-          <form action="#">
-            <input type="text" placeholder="Search.." name="search" />
-            <button type="submit"><i class="fa fa-search"></i></button>
-          </form>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link href="styles.css" rel="stylesheet"> -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" >
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <title>Pet Connect</title>
+    <style>
```

Comment:
This should go in the CSS file: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns